src/HotChocolate/Fusion/src/Core/Execution/SelectionData.cs

```diff
@@ -19,6 +19,13 @@ private SelectionData(JsonResult[] multiple)
         HasValue = true;
     }
 
+    private SelectionData(bool hasError)
+    {
+        HasError = hasError;
+    }
+
+    public bool HasError { get; }
```

Comment:
This is not correct, as there can be both successful and unsuccessful data as part of a single SelectionData.